StackExchange.Redis/StackExchange/Redis/PhysicalBridge.cs

```diff
@@ -70,6 +70,8 @@ public enum State : byte
 
         public bool IsConnected => state == (int)State.ConnectedEstablished;
 
+        public bool IsConnecting => state == (int)State.ConnectedEstablished || state == (int)State.Connecting;
```

Comment:
This one doesn't make sense to me, why is `state == (int)State.ConnectedEstablished` considered "connecting"? It's already the case for IsConnected above, I'd expect this to only return `state == (int)State.Connecting`

Reply:
The rationale was that if it's not yet `Connected`, it is still `Connecting`.  However if it's easier to understand I'd be happy to adjust, as long as there is a mechanism to determine that the multiplexor is currently in the process of connecting.

Reply:
@tapmantwo Not sure I follow, are you saying `state == (int)State.ConnectedEstablished` is "not yet connected"? That's the part I'm having a disconnect on, as we read that as "IsConnected" above

Reply:
Oh my, that is a good spot!  It was supposed to be State.ConnectedEstablishing.  I will update now :)